package.json

```diff
@@ -23,7 +24,7 @@
     "shift-ast": "git://github.com/shapesecurity/shift-ast-js.git#591c3376fde127c6bdf6d5c61c00ab309af456c3"
   },
   "devDependencies": {
-    "6to5": "^1.14.14",
+    "6to5": "^3.4.1",
```

Comment:
We will lock to specific versions from now on, to help ensure deterministic builds. Remove the caret and submit this change as a separate PR..